InventoryManagementSystem/Product.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+namespace InventoryManagementSystem
+{
+	public class Product
+	{
+        public string Name { get; set; }
+        public double Price { get; set; }
+        public int Quantity { get; set; }
+
+      
+
+		public Product(string name, double price, int quantity)
+		{
+			Name = name;
+			Price = price;
+			Quantity = quantity;
+		}
```

Comment:
please format the code 
( as there are extra lines and unneeded leading spaces)
if you search you will find that your Ide can do that automatically